packages/examples/react/src/app.tsx

```diff
@@ -77,82 +120,27 @@ export class ReactMonacoEditor extends React.Component<EditorProps> {
             // install Monaco language client services
             MonacoServices.install();
 
-            this.createWebSocket();
+            createWebSocket(url);
 
-            this.resizeEvent = () => this.editor?.layout();
-            window.addEventListener('resize', this.resizeEvent);
-        }
-    }
+            const resizeEvent = () => editorRef.current!.layout();
+            window.addEventListener('resize', resizeEvent);
```

Comment:
What is the issue with `automaticLayout`?

Reply:
Oh boy, good point. 🙈 I wasn't aware this is possible or I seem to have forgotten about. Now you asked the question, I tested it and it works nicely.

Reply:
I think on earlier version of monaco it wasn't great as they were polling, but for some versions now they use a ResizeObserver

Reply:
👍  Good to know. I am missing some history here.